ironfish-cli/src/commands/service/estimate-fee-rates.ts

```diff
@@ -76,19 +76,19 @@ export default class EstimateFees extends IronfishCommand {
               timestamp: new Date(),
               fields: [
                 {
-                  name: `fee_rate_low`,
+                  name: `fee_rate_slow`,
```

Comment:
Making a note here that we need to update `ironfish-api` to add these to the whitelist and remove the old ones

Reply:
It looks like the api whitelists everything from `fee_rate_estimate` measurement so idk that we need to update these

Reply:
https://github.com/iron-fish/ironfish-api/blob/260efa7c243f20e683136b4443b40a7c151fc734/src/telemetry/telemetry.controller.ts#L80